Makefile

```diff
@@ -226,7 +226,9 @@ test:
 	export DYLD_LIBRARY_PATH="${DYLD_LIBRARY_PATH}:${LOCAL_DIR}/lib" && \
 	export LOG_LEVEL=DEBUG && \
 	export RUST_BACKTRACE=1 && \
-	cargo test --no-default-features --features "${ENABLE_FEATURES}" --all --exclude tests --exclude cdc ${EXTRA_CARGO_ARGS} -- --nocapture && \
+	cargo test --no-default-features --features "${ENABLE_FEATURES}" --all --exclude tests --exclude \
```

Comment:
why is cdc also excluded?

Reply:
No idea. It was excluded before so I left it excluded

Reply:
Because cdc and tests are not included in features `prost/protobuf-codec` in src/Cargo.toml.